federation/pkg/typeadapters/secret.go

```diff
@@ -27,10 +28,13 @@ import (
 	kubeclientset "k8s.io/kubernetes/pkg/client/clientset_generated/clientset"
 )
 
-const SecretKind = "secret"
+const (
+	SecretKind           = "secret"
```

Comment:
better to use camel case for SecretKind, it is the convention used for kube object kind.